registration-processor-default.properties

```diff
@@ -483,14 +484,14 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.version=0.9
 mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1=${mosip.biosdk.default.service.url}
 
 ## Credential requestor
-mosip.registration.processor.credentialtype=euin
-mosip.registration.processor.encrypt=false
-mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
+#mosip.registration.processor.credentialtype=euin
+#mosip.registration.processor.encrypt=false
+#mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator
 
-mosip.registration.processor.credential.partner-profiles=registration-processor-credential-partners.json
-mosip.registration.processor.credential.default.partner-ids=digitalcardPartner,opencrvsPartner
-mosip.registration.processor.credential.conditional.partner-id-map={'printPartner':'{"14023"} contains postalCode'}
-mosip.registration.processor.credential.conditional.no-match-partner-ids=printPartner
+#mosip.registration.processor.credential.partner-profiles=registration-processor-credential-partners.json
+#mosip.registration.processor.credential.default.partner-ids=digitalcardPartner,opencrvsPartner,digitalCardInsurancePartner
+#mosip.registration.processor.credential.conditional.partner-id-map={'printPartner':'{"14023"} contains postalCode'}
+#mosip.registration.processor.credential.conditional.no-match-partner-ids=printPartner,digitalCardInsurancePartner
```

Comment:
Remove these duplicate properties and put correct values in the properties where ever it is mentioned.

Reply:
Done